src/main/java/Control/Scenario/UC_NewProject.java

```diff
@@ -0,0 +1,173 @@
+package Control.Scenario;
+
+import Control.Connect.DbProvider;
+import Model.Database.Interaction.I_Project;
+import Model.Database.Interaction.I_ProjectUser;
+import Model.Database.Interaction.I_User;
+import Model.Database.Support.CustomLogs;
+import Model.Database.Tables.Table.T_Project;
+import Model.Database.Tables.Table.T_Project_user;
+import Model.Database.Tables.Table.T_User;
+import Model.Web.JsonResponse;
+import Model.Web.Specific.ProjectCreation;
+import View.Support.CustomExceptions.AlreadyExistsException;
+import View.Support.CustomExceptions.ProjectCreationException;
+import org.apache.commons.validator.routines.EmailValidator;
+
+import javax.servlet.http.HttpServletResponse;
+import javax.validation.constraints.NotNull;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Dictionary;
+import java.util.Hashtable;
+import java.util.List;
+
+public class UC_NewProject {
+    private DbProvider db;
+
+    public UC_NewProject(@NotNull DbProvider dbProvider) {
+        this.db = dbProvider;
+    }
+
+    public final @NotNull JsonResponse createNewProject(@NotNull final ProjectCreation projectCreation) {
+        JsonResponse jsonResponse = new JsonResponse();
+
+        try {
+            db.beforeSqlExecution(true);
+
+            // do not create project that already exists
+            if (getProjectByName(projectCreation.getProject_name()) != null) {
+                jsonResponse.setMessage("Project with this  name already exists.");
```

Comment:
je to hovadina, ale tu aj v riadku nizsie mas dve medzery medzi this a name

Reply:
Môžeš zmeniť ked budeš kodiť dalej